script.js

```diff
@@ -0,0 +1,202 @@
+// We'll pre-populate this array with a couple objects just so it's not undefined if your internet connection isn't working properly.
+
+/*let arrayOfTodos = [
+  {
+  "userId": 14,
+  "id": 1,
+  "title": "delectus aut autem",
+  "completed": false
+},
+{
+  "userId": 20,
+  "id": 2,
+  "title": "delectus aut autem",
+  "completed": false
+}] */
+
+
+const fetchTodos = () => {
+  fetch('https://jsonplaceholder.typicode.com/todos')
+    .then((response) => {
+      //console.log ('OG response:', response)
+      return response.json()
+    })
+
+    .then((json) => {
+      console.log('OG json:', json)
+      arrayOfTodos = json
+    })
+}
+
+const logTodos = () => {
+  console.log(arrayOfTodos)
+}
+
+//Make button work with JSON auto-populating list
+
+const populateTodos = () => {
+  // var thirdObj = todos[3]
+  // var status = thirdObj.completed
+  console.log(status)
+  createSublist(container, arrayOfTodos);
```

Comment:
problem here is you're passing in a container value and an arrayOfTodos. container doesn't seem to be defined anywhere:` let container = ""` and arrayOfTodos is commented out at the top of the page. so I'd bring that back. and then when you hit fetchTodos, line 27 will set the value to it.